lib/l10n/app_de.arb

```diff
@@ -301,6 +301,10 @@
   "@showCoverAsPlayerBackgroundSubtitle": {},
   "hideSongArtistsIfSameAsAlbumArtists": "Verstecke Titel-Künstler falls derselbe wie Album-Künstler",
   "@hideSongArtistsIfSameAsAlbumArtists": {},
+  "showArtistsTopSongs": "Meistgehörte Lieder bei Künstler anzeigen",
+  "@showArtistsTopSongs": {},
+  "showArtistsTopSongsSubtitle": "Ob oder nicht die meistgehörte Lieder bei Künstlern angezeigt werden sollen.",
```

Comment:
"Ob die meistgehörten Lieder eines Künstlers angezeigt werden sollen, oder nicht."  
or maybe just
"Sollen die meistgehörten Lieder eines Künstlers (auf der Künstler-Seite) angezeigt werden?"

Reply:
I would prefer the first one without the "oder nicht", I thought maybe that's just a Germany-exclusive colloquialism that I didn't know about, and I followed the other translations there. I'll change all occurrences as I hate the phrasing